api/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require(`${__dirname}/controllers/topics.controllers`)
+const { handlePsqlErrors, handleCustomErrors, handleInternalErrors} = require(`${__dirname}/errors/errors`)
+
+app.get("/api/topics", getTopics)
+
+app.use(handlePsqlErrors)
+
```

Comment:
we should only be writing the code that meets the requirements of the pr, we aren't dealing with psql errors yet so we don't need to account for this